tron/core/jobrun.py

```diff
@@ -227,6 +228,7 @@ def handle_action_run_state_change(self, action_run: ActionRun, event):
 
         # propagate all state changes (from action runs) up to state serializer
         self.notify(self.NOTIFY_STATE_CHANGED)
+        metrics.meter(f'tron.actionrun.{event}')
```

Comment:
This is cool.
What do you think about actually putting individual namespace.job.action names in here?
We would explode the metrics to "a thousand or so", but it means we could action runtimes on our charts. Wouldn't that be cool?

Reply:
I **could** put something in `_done` and emit the difference between start_time and end_time for each action. But one data point/day doesn't seem like the best fit for these metrics, which use exponentially decaying samples and stuff.

We can already query the API and get the most recent duration for an action. Maybe in the future we could add a different API endpoint that provides actual stats from the state file for jobs and actions.